pylab/website/tests/test_about_page.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+
+from django_webtest import WebTest
+from django.contrib.auth.models import User
+
+from pylab.core.models import Event
+
+
+class AboutPageTests(WebTest):
+
+    def setUp(self):
+        self.user = User.objects.create(username='u1')
+
+    def test_no_events_on_about_page(self):
+        resp = self.app.get('/about/')
+        self.assertEqual(resp.status_int, 200)
+        self.assertTrue(b'No events yet.' in resp.content)
+
+    def test_event_list_on_about_page(self):
+        Event.objects.create(
+            author=self.user,
+            starts=datetime.datetime(2015, 9, 3),
+            ends=datetime.datetime(2015, 9, 3),
+            title='Test title',
+            osm_map_link='http://openstreetmap.org/',
+            description='Test description',
+        )
```

Comment:
User [UserFactory](https://github.com/python-dirbtuves/website/blob/master/pylab/core/factories.py#L10) instead of creating Event instance directly from model.

[Factory boy](https://factoryboy.readthedocs.org/en/latest/) factories allows you to control model creation in tests in a more flexible way. For example if you change structure of you models, you will be able to update you factories instead of changing all tests (of course this is not always the case, but most of the time, factories really help to manage you test fixtures especially when data structure becomes more complex).